src/Umbraco.Cms.Integrations.Web/Umbraco.Cms.Integrations.Web.AuthorizationHub/Configuration/SemrushSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿
```

Comment:
Do we still need this project? If not, best to remove.

Reply:
Item addressed.